features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -280,3 +280,47 @@ Scenario: Supplier asks a clarification question
   And I click 'Ask question'
   Then I see a success banner message containing 'Your question has been sent.'
 
+@sign-framework-agreement-call-to-action
+Scenario: Supplier can see sign framework agreement call to action
+  Given I have a live digital outcomes and specialists framework
+  And that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And that supplier has filled in their application but not submitted it
+  When I click the 'View your account' link
+  Then I see the 'You must sign the framework agreement to sell these services' link
+
+@opportunities-overview
+Scenario: Supplier can see the link to the opportunities overview
+  Given I have a live digital outcomes and specialists framework
```

Comment:
There's a lot of repetition of 
```
Given I have a live digital outcomes and specialists framework
And that supplier is on that framework
And that supplier has a service on the digital-specialists lot
And I have a live digital-specialists brief
And that supplier has filled in their application but not submitted it
```

Might be worth moving it to a setup step rather than repeating in each scenario?